philipmeneghini/InstaConnect
# Edit Profile and Create Post Functionality

Buttons follow, profile, edit profile and create post on profile pages now work. All these buttons with the exception of the follow and unfollow button open in a Modal window in which you can upload photos and view/edit profile information. At this time this is only supported for `.jpg` files.

![ProfilePageButtons](https://github.com/philipmeneghini/InstaConnect/assets/63440066/5b01cef6-0fee-4e86-ac30-cafdecb457ed)

**Testing**

- Click each button and verify that it opens the correct window
- Verify that edited profile values appear in the database and in application refresh
- Check that email service and s3 uploads are working correctly
- Test in multiple sized windows to check for css irregularity
